dom.bs

```diff
@@ -1280,6 +1321,20 @@ for discussion).
 
  <li><p>Set <var>event</var>'s <a for=Event>path</a> to the empty list.
 
+ <li>
+  <p>If <var>event</var> is a {{MouseEvent}} object, <var>event</var>'s {{Event/type}} attribute is
```

Comment:
Maybe factor out the condition into a variable? I'm pretty sure an implementation would, and it makes it very clear that there's no case where *activationTarget* has changed in the interim to consider.